godot_communicator.py

```diff
@@ -7,6 +7,7 @@
 import time
 
 velocity = 0
+time_stamp = time.time()
```

Comment:
I'm not a fan of this solution of adding yet another global variable. In the end the godot communicator itself shouldn't care about the current time to do its communication (it also shouldn't care about velocity but we'll fix that later).

Reply:
not to forget this value is updated by 2 functions on different threads making the value entirely unpredictable. Maybe discuss with Tom if we wanna set a fixed update rate of 60 times per second, which would solve most of the problems regarding the current solution.